input/haleu/analysis/df_to_csv.py

```diff
@@ -17,8 +20,8 @@ def get_transactions_multiple_outputs():
 
 
 def get_transactions_single_output():
-    scenario = 'current'
-    outfile = f'../outputs/united_states_2020.sqlite'
+    scenario = 'united_states_2020'
+    outfile = f'../outputs/{scenario}.sqlite'
```

Comment:
Wouldn't it be more efficient to have the `scenario` variable be parameterized? Just something to think about. It seems like this gets changed every few PRs or so. It may not be worth it as you'd need to set up a different place to store that information, and have a way to pass it to your python scripts.